install_template/templates/platformBase/_deploymentConstants.njk

```diff
@@ -33,6 +33,7 @@
     "Ubuntu 18.04": "ubuntu_18",
     "Ubuntu 20.04": "ubuntu_20",
```

Comment:
Another Ubuntu 20 referenc

Reply:
We have to keep this one for now as this file concerns a lot of products and not just the DMS. We'll request ubuntu 20 removal for all remaining docs in a separate PR.